Docker.DotNet.BasicAuth/Docker.DotNet.BasicAuth.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="**\*.cs" />
```

Comment:
If we use the latest bits from the SDK, we can remove it however, it is broken. Visual Studio don't work with it, just the CLI. I'm looking at it with the proper team here https://github.com/dotnet/sdk/issues/604

The version I added on AppVeyor scripts is the `preview4` which is the *stable* version of the MSBuild sdk for .net core. And that require this globbing. 

When they fix that issue (scheduled for the next release of the SDK probably) I can remove it and update the build script.